steps/synopsys-pt-timing-signoff/pt.tcl

```diff
@@ -9,22 +9,23 @@ set report_default_significant_digits 3
 ##################################################################
 #    Constraint Analysis Section
 ##################################################################
-check_constraints -verbose > check_constraints.report
+check_constraints -verbose > reports/check_constraints.report
 
 ##################################################################
 #    Update_timing and check_timing Section                      #
 ##################################################################
 
 update_timing -full
-check_timing -verbose > check_timing.report
+check_timing -verbose > reports/check_timing.report
 
 ##################################################################
 #    Report_timing Section                                       #
 ##################################################################
-report_global_timing > report_global_timing.report
-report_clock -skew -attribute > report_clock.report 
-report_analysis_coverage > report_analysis_coverage.report
-report_timing -crosstalk_delta -slack_lesser_than 1000.0 -max_paths 100 -pba_mode exhaustive -delay min_max -nosplit -input -net > report_timing_pba.report
+report_global_timing > reports/report_global_timing.report
```

Comment:
And add a new line here. Let's just make all the standard nodes coding convention similar.